packages/block-library/src/calendar/block.json

```diff
@@ -40,6 +40,18 @@
 		},
 		"interactivity": {
 			"clientNavigation": true
+		},
+		"__experimentalBorder": {
+			"radius": true,
+			"color": true,
+			"width": true,
+			"style": true,
+			"__experimentalDefaultControls": {
+				"radius": true,
+				"color": true,
+				"width": true,
+				"style": true
+			}
```

Comment:
``` suggestion
```

The current thinking on whether border controls should show by default is that they should be optional for niche cases. 

The best indicator for displaying border controls by default is whether the dimension controls are also displayed by default. With no padding or margin support on this block, I'd say we should hide these controls by default.